course7/homeworks/mart/complex/complex_linked_list.cc

```diff
@@ -0,0 +1,33 @@
+#include "complex_linked_list.hh"
+
+bool ComplexLinkedList::empty() const
+{
+	return mSize == 0;
+}
+
+std::size_t ComplexLinkedList::size() const
+{
+	return mSize;
+}
+
+Complex& ComplexLinkedList::back()
```

Comment:
these one-lines should really be in the header file. Such a small functions benefit from inlining the most.